<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateTable</class>
 <widget class="QMainWindow" name="CreateTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Graph</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pbt_creat_graph">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>330</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Изобразить граф</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>351</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbt_back">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Назад</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbt_symmetric">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Симмитричное заполнение таблицы</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbt_save_to_csv">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>370</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Сохранить в csv</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>351</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
